depends/Makefile

```diff
@@ -156,7 +156,11 @@ $(host_arch)_$(host_os)_native_toolchain?=$($(host_os)_native_toolchain)
 include funcs.mk
 
 binutils_path=$($($(host_arch)_$(host_os)_native_binutils)_prefixbin)
+ifeq ($(strip $(FORCE_USE_SYSTEM_CLANG)),)
```

Comment:
Please add FORCE_USE_SYSTEM_CLANG to the options in README.md (preferably with an expert disclaimer).

Reply:
See: de4fedb6c3a45e6a23f1bccd28045d76b5830afc